liwords-ui/src/tournament/pairings.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactNode } from 'react';
+import { useTournamentStoreContext } from '../store/store';
+import { Table, Tag } from 'antd';
+import { Division, SinglePairing } from '../store/reducers/tournament_reducer';
+
+const usernameFromPlayerEntry = (p: string) =>
+  p.split(':').length > 0 ? p.split(':')[1] : 'Unknown player';
```

Comment:
not a big deal but should avoid splitting twice. I'll merge anyway, we can handle it after.